api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupTXNEW.java

```diff
@@ -281,7 +250,7 @@ public ReportDefinition constructReportDefinition() {
 		// Obtain patients notified to be on TB treatment 
 		SqlCohortDefinition notifiedToBeOnTbTreatment = new SqlCohortDefinition();
```

Comment:
Please move this cohort query into the `SqlCohortQueries.java`

Reply:
[Done](https://github.com/esaude/openmrs-module-eptsreports/commit/4f7f692c5bc44b5843f3fb98a8c8b7510b676923)